src/core_modules/capture-core/src/components/ColumnSelector/ColumnSelector.container.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { connect } from 'react-redux';
+import ColumnSelector from './ColumnSelector.component';
+import { updateWorkinglistOrder } from './actions/ColumnSelector.actions';
+
+const mapStateToProps = (state: Object) => ({
+    workingListColumnOrder: state.workingListsColumnsOrder.main,
```

Comment:
please use the columns Object from the eventList. This object already has the headers/texts so no need to retrieve them again in DragDropList.component.